package.json

```diff
@@ -16,13 +16,13 @@
     "test": "npm run lint && npm run cover"
   },
   "dependencies": {
-    "camelcase": "^3.0.0",
-    "http-status": "^1.4.2"
+    "camelcase": "^3.0.0"
   },
   "devDependencies": {
     "chai": "^4.2.0",
     "coveralls": "^3.1.0",
     "eslint": "^7.0.0",
+    "http-status": "^1.4.2",
```

Comment:
Keeping this as a dev dependency since it's used in some test assertions.

Reply:
can add a "will deprecate" note in README to ask developers to move away from using status message for dev test, or require `http-status` on their own.

Reply:
The tests would pass if I removed this entirely and just used the subset, but it seemed nice to just assert against the same library.

https://github.com/yahoo/fumble/blob/master/tests/unit/HttpError.js#L44

```js
expect(error).to.have.property('message', STATUS_CODES[400]);
```

Thinking more though, it would just make sense to remove it entirely - I'll update this.